templates/dashboard/setting.html

```diff
@@ -584,6 +584,42 @@
           </div>
         </div>
         <!-- END One-click subscribe -->
+        <div class="card" id="unsub-behaviour">
```

Comment:
can we add the new option into this existing settings instead?
<img width="972" alt="image" src="https://user-images.githubusercontent.com/3167780/177947284-eceb5718-1908-46b0-bbc4-c631974bdb3b.png">

Reply:
Ok. Will do.

Reply:
![image](https://user-images.githubusercontent.com/663855/178541330-940c48e5-c9c0-47af-927e-91539bd316cd.png)

Reply:
Done